pulpcore/cli/common/openapi.py

```diff
@@ -215,11 +215,24 @@ def parse_response(self, method_spec: Dict[str, Any], response: requests.Respons
         if response.status_code == 204:
             return "{}"
 
+        if response.status_code in (301, 302):
+            raise OpenAPIError(
+                _("Received redirect to '{url}'.").format(url=response.headers["location"])
```

Comment:
I think, we should be more explicit that the user should do something about it.

Reply:
Agreed. I couldn't think of a good message. Any suggestions?